### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by guruxp**
Updated article to include the 10aux archive limit, licensing details, content that will be moved, and what will happen with Teams chat data

### CODE REVIEW

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This article describes the process for cross-tenant mailbox moves and provides g
 
 ## Licensing
 
-Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (onetime fee). Please contact your Microsoft account team for details.
+Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (onetime fee) and can be assigned either on the source or target user object. This license also covers the OneDrive for Business migration. Please contact your Microsoft account team for details.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers OneDrive for Business migration. Please contact your Microsoft account team for details.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 2]
```diff
@@ -396,7 +396,11 @@ The meetings will move, however the Teams meeting URL does not update when items
 
 ### Does the Teams chat folder content migrate cross-tenant?
 
-No, the Teams chat folder content does not migrate cross-tenant. When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox (email, contacts, calendar, tasks, and notes) is migrated.
+No, the Teams chat folder content does not migrate cross-tenant. However, once the mailbox has been migrated cross-tenant, the Teams chat folder content will be available for source tenant admins to search and export using a content search.
+
+### What content is migrated cross-tenant then?
+
+When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox, also known as Top Of Information Store (email, contacts, calendar, tasks, and notes) and recoverable items subfolders except SubstrateHolds is migrated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and recoverable items subfolders except SubstrateHolds are migrated.  
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 3]
```diff
@@ -580,7 +584,7 @@ However, there are some things you need to be aware of.
...
 
-- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive – meaning more than one archive mailbox, the feature is unable to migrate the additional archives and should fail.
+- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there's no more than 10 aux archives, if there's more than 10 aux archives the migration will fail.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there are no more than 10 aux archives. If there are more than 10 aux archives, the migration will fail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @guruxp, kindly commit the suggested changes. Many thanks.

**Comment by guruxp** [comment]
@georgiahuggins mind taking a look at this one please?

**Comment by guruxp** [comment]
@JohanFreelancer9  done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by guruxp** [comment]
@kelleyvice-msft any plans on merging this during the week?

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 4]
```diff
@@ -396,7 +396,11 @@ The meetings will move, however the Teams meeting URL does not update when items
 
 ### Does the Teams chat folder content migrate cross-tenant?
 
-No, the Teams chat folder content does not migrate cross-tenant. When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox (email, contacts, calendar, tasks, and notes) is migrated.
+No, the Teams chat folder content does not migrate cross-tenant. However, once the mailbox has been migrated cross-tenant, the Teams chat folder content will be available for source tenant admins to search and export using a content search.
+
+### What content is migrated cross-tenant then?
+
+When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and recoverable items subfolders except SubstrateHolds are migrated.
```
Comment (tweekerz) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: Update this line: 
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges.

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 5]
```diff
@@ -396,7 +396,11 @@ The meetings will move, however the Teams meeting URL does not update when items
 
 ### Does the Teams chat folder content migrate cross-tenant?
 
-No, the Teams chat folder content does not migrate cross-tenant. When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox (email, contacts, calendar, tasks, and notes) is migrated.
+No, the Teams chat folder content does not migrate cross-tenant. However, once the mailbox has been migrated cross-tenant, the Teams chat folder content will be available for source tenant admins to search and export using a content search.
+
+### What content is migrated cross-tenant then?
```
Comment (tweekerz) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: Let's remove the then and move this above the chat folder and remove the Then.

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ This article describes the process for cross-tenant mailbox moves and provides g
 
 ## Licensing
 
-Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (onetime fee). Please contact your Microsoft account team for details.
+Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers OneDrive for Business migration. Please contact your Microsoft account team for details.
```
Comment (tweekerz) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: Update this "This license also covers OneDrive for Business migration link to content" to add the link to the documentation for this https://learn.microsoft.com/microsoft-365/enterprise/cross-tenant-onedrive-migration

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 7]
```diff
@@ -580,7 +584,7 @@ However, there are some things you need to be aware of.
...
 
-- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive – meaning more than one archive mailbox, the feature is unable to migrate the additional archives and should fail.
+- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there are no more than 10 aux archives. If there are more than 10 aux archives, the migration will fail.
```
Comment (tweekerz) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: We already have this posted in this document under Do auto-expanded archive mailboxes move?
Yes, if the user in source has auto-expanding archives enabled and has additional auxiliary archives, cross-tenant mailbox migration will work. We support moving users that have no more than 12 auxiliary archive mailboxes.

So we just need to remove this line completly

**Review by tweekerz — CHANGES_REQUESTED** [review]
Reviewed this with the PM of the feature and added some notes

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md (hunk 4 above)
Comment (guruxp) [reply microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: Updated to: 
"When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges is migrated. "

Although I still think it's worth mentioning SubstrateHolds is dropped from there.

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md (hunk 5 above)
Comment (guruxp) [reply microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: done

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md (hunk 6 above)
Comment (guruxp) [reply microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: done

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md (hunk 7 above)
Comment (guruxp) [reply microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: done

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 8]
```diff
@@ -43,7 +43,7 @@ When a mailbox is migrated cross-tenant with this feature, only user visible con
 
 ## Licensing
 
-Cross Tenant User Data Migration is available as an add-on to the Microsoft 365 subscription plans below and is only available to customers with an active Enterprise Agreement customer. User licenses are per migration (onetime fee). Contact your Microsoft account team for details.
+Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers [OneDrive for Business migration](https://learn.microsoft.com/microsoft-365/enterprise/cross-tenant-onedrive-migration). Please contact your Microsoft account team for details.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers [OneDrive for Business migration](/microsoft-365/enterprise/cross-tenant-onedrive-migration). Contact your Microsoft account team for details.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 9]
```diff
@@ -383,9 +383,13 @@ Get-MailUser -Identity <Migrate Mail User> | Enable-RemoteMailbox
 
 While Teams meetings are moved, the meeting URL isn't updated when items migrate cross-tenant. Since the URL will be invalid in the target tenant, you must remove and recreate Teams meetings.
 
+### What content is migrated cross-tenant?
+
+When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions and Purges is migrated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges is migrated.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@guruxp, kindly commit the minor suggested changes. Many thanks.

**Comment by guruxp** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 10]
```diff
@@ -383,9 +383,13 @@ Get-MailUser -Identity <Migrate Mail User> | Enable-RemoteMailbox
 
 While Teams meetings are moved, the meeting URL isn't updated when items migrate cross-tenant. Since the URL will be invalid in the target tenant, you must remove and recreate Teams meetings.
 
+### What content is migrated cross-tenant?
+
+When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges is migrated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@guruxp. please commit the suggested change to the updated content. Many thanks.

**Comment by guruxp** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by guruxp** [comment]
@JohanFreelancer9 any ETA on when this will be merged please?

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 any ETA on when this will be merged please?

@yogkumgit, could you please help? Thanks.

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder.

**Comment by guruxp** [comment]
@JohanFreelancer9 @yogkumgit @kelleyvice-msft any progress on this one please?

**Comment by guruxp** [comment]
@JohanFreelancer9 @yogkumgit please?

**Comment by guruxp** [comment]
@georgiahuggins any way to unblock this?

**Comment by tweekerz** [comment]
Hi @***@***.***>, it looks like @***@***.***> is on LOA. Can you find someone who can approve this? I have an update that is pending this change.

Rob Whaley
Sr. Support Escalation Engineer,
Exchange, Outlook, and SCI Beta Engineer and Global Content & Readiness Lead
Empowering, Helping, and Advising Customers

***@***.***
***@***.******@***.***?anonymous&ep=signature>Book time to meet with ***@***.***?anonymous&ep=signature>

**Comment by JohanFreelancer9** [comment]
> @georgiahuggins any way to unblock this?

@yogkumgit

**Comment by yogkumgit** [comment]
@tiburd Please help. Thanks

**Comment by tiburd** [comment]
@georgiahuggins 
Hi, 
Can you please review/approve these changes?
Thank you!

**Comment by tiburd** [comment]
approved by Georgia via email. Merging.

**Pull request merged**